src/pudl/glue/epacems_unitid_eia_plant_crosswalk.py

```diff
@@ -0,0 +1,113 @@
+"""Extract, clean, and normalize the EPACAMD-EIA crosswalk.
```

Comment:
Should make sure we rename this module to match whatever the newly selected name for the table is.

Reply:
Renamed. Important to note that we still need to fix the name in the scraper and archiver before we can fix the names in the pudl extractor.